src/components/cron.ts

```diff
@@ -30,3 +32,13 @@ export const createSuggestionCron = (client: CommandoClient): CronJob =>
       }
     }
   });
+
+// Gives Codey coin bonus to those on the interviewer list at 1:00 am UTC
+export const createBonusInterviewerListCron = (): CronJob =>
+  new CronJob('0 0 1 * * *', async function () {
+    const activeInterviewers = await getInterviewers(null);
+    activeInterviewers.forEach(async (interviewer) => {
+      const bonus = coinBonusMap.get(BonusType.InterviewerList);
```

Comment:
what if `coinBonusMap` doesn't contain the desired bonus type? can we do a null check so we don't need non null assertions? I feel like non null assertions should be used sparingly.